src/core/cgroup.c

```diff
@@ -293,8 +293,13 @@ static int whitelist_device(const char *path, const char *node, const char *acc)
         assert(acc);
 
         if (stat(node, &st) < 0) {
-                log_warning("Couldn't stat device %s", node);
-                return -errno;
+                /* path starting with "-" must be silently ignored */
+                if (startswith(node, "-"))
+                        return 0;
+                else {
+                        log_warning("Couldn't stat device %s", node);
```

Comment:
if the first if branch is just a "return 0", we dont need the second branch, and can just place it directly, as-is. After all if we can avoid extra indentation that's a good thing!